pkg/internal/ibmblockcsi/static_resource_generator.go

```diff
@@ -25,6 +25,28 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	snapshotStorageApiGroup					string = "snapshot.storage.k8s.io"
```

Comment:
consider using spaces instead of tabs

Reply:
+1.
Please use gofmt to format your code

Reply:
fixed